src/Umbraco.Cms.Api.Management/ViewModels/Template/TemplateViewModel.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace Umbraco.Cms.Api.Management.ViewModels.Template;
+
+public class TemplateViewModel : TemplateModelBase
+{
+    public Guid Key { get; set; }
+
+    public bool IsMasterTemplate { get; set; }
+
+    public string? VirtualPath { get; set; }
```

Comment:
In what cases should this not be calculated from the alias/name?

Reply:
They're removed now - apparently they're used purely for presentation purposes, and at this time we don't know if they will be relevant in the new backoffice.